spec/models/user_sign_in_event_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'yaml'
+require 'rails_helper'
+
+RSpec.describe UserSignInEvent, type: :model do
+  include Utilities::Configuration::Settings
+
+  user_id = SecureRandom.uuid
+  let(:user_id) { SecureRandom.uuid }
+   let(:event) { UserSignInEvent.create(user_id: user_id) }
+
+  context '#create' do
+    it 'creates a valid event which contains hard-coded data' do
+      expect(event.data).to include(
+        'user_id'
+      )
+      expect(event.data['user_id']).to eq(user_id)
```

Comment:
Do we need to check it twice?

Reply:
yes... They check different pits of the data